Jenkinsfile_CNP

```diff
@@ -78,5 +87,11 @@ withPipeline(type, product, component) {
   installCharts()
   disableLegacyDeployment()
   enableSlackNotifications(channel)
-  loadVaultSecrets(secrets)
+
+  onPR() {
+    loadVaultSecrets(prSecrets)
+  }
+  onNonPR() {
+    loadVaultSecrets(nonPrSecrets)
+  }
```

Comment:
what are those changes related to? Not surname ocr field I assume? can it go to a separate PR with explanation?